src/Paket/Program.fs

```diff
@@ -776,6 +794,7 @@ let handleCommand silent command =
     | GenerateLoadScripts r -> processCommand silent generateLoadScripts r
     | GenerateNuspec r -> processCommand silent generateNuspec r
     | Why r -> processCommand silent why r
+    | Restriction r -> processCommand silent restriction r
```

Comment:
@forki I can remove that from the PR if you don't like it, the rest are bug-fixes

Reply:
no it's fine, but needs docs

Reply:
That's the question, I saw it more like a debug helper for us, but we can make it a feature (honestly I doubt if the regular user can do something with the output)

Reply:
I'll add some lines :)